tests/conftest.py

```diff
@@ -19,6 +19,24 @@
 edit   history  py    quit      shell  unalias
 """
 
+BASE_HELP_VERBOSE = """
```

Comment:
Thanks for adding the unit tests.  They should really help with maintainability.